documentation/docs/pipelines/abapEnvironment/introduction.md

```diff
@@ -0,0 +1,92 @@
+# ABAP Environment Pipeline
+
+The goal of the ABAP Environment Pipeline is to enable Continuous Integration for the SAP Cloud Platform ABAP Environment, also known as Steampunk.
+In the current state, the pipeline enables you to pull your Software Components to specifc systems and perform ATC checks. The following steps are performed:
+
+* Create an instance of the SAP Cloud Platform ABAP Environment
+* Configure the Communication Arrangement SAP_COM_0510
+* Pull Git repositories / Software Components to the instance
```

Comment:
Maybe we could write "Pull git repositories / Software Components to the SAP Cloud Platform ABAP Environment System" or would this be too many mentions of the term SAP Cloud Platform ABAP Environment?

Reply:
> would this be too many mentions of the term SAP Cloud Platform ABAP Environment?
I would say so, yes. But I guess this is personal taste.